app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/ManageApplications.scala

```diff
@@ -17,46 +17,42 @@
 package uk.gov.hmrc.thirdpartydeveloperfrontend.controllers
 
 
-import uk.gov.hmrc.thirdpartydeveloperfrontend.config.ApplicationConfig
-import uk.gov.hmrc.thirdpartydeveloperfrontend.config.ErrorHandler
-import javax.inject.{Inject, Singleton}
 import play.api.libs.crypto.CookieSigner
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.apiplatform.modules.uplift.services.UpliftLogic
+import uk.gov.hmrc.thirdpartydeveloperfrontend.config.{ApplicationConfig, ErrorHandler}
+import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.controllers.ManageApplicationsViewModel
 import uk.gov.hmrc.thirdpartydeveloperfrontend.service._
 import views.helper.EnvironmentNameService
 import views.html._
 
+import javax.inject.{Inject, Singleton}
 import scala.concurrent.ExecutionContext
-import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.controllers.ManageApplicationsViewModel
-import uk.gov.hmrc.apiplatform.modules.uplift.services.UpliftLogic
 
 @Singleton
-class ManageApplications @Inject() (
-    val errorHandler: ErrorHandler,
-    val sessionService: SessionService,
-    val cookieSigner: CookieSigner,
-
-    appsByTeamMember: AppsByTeamMemberService,
-    upliftLogic: UpliftLogic,
-    
-    manageApplicationsView: ManageApplicationsView,
-    addApplicationSubordinateEmptyNestView: AddApplicationSubordinateEmptyNestView,
-    mcc: MessagesControllerComponents
-)(implicit val ec: ExecutionContext, val appConfig: ApplicationConfig, val environmentNameService: EnvironmentNameService)
-   extends LoggedInController(mcc) {
+class ManageApplications @Inject()(
```

Comment:
Can you sort the formatting please.  Really object to things been this indented

Reply:
will do